packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -235,7 +238,7 @@ export class AtomicSearchInterface {
   /**
    * Initializes the connection with the headless search engine using options for `accessToken` (required), `organizationId` (required), `renewAccessToken`, and `platformUrl`.
    */
-  @Method() public initialize(options: InitializationOptions) {
+  @Method() public async initialize(options: InitializationOptions) {
```

Comment:
Why the async keyword if we do not use await in the body of the function ? 

It was returning a promise anyway before, no ?